### DISCUSSION TITLE
rAudio 20240218 issues - invalid card number 'scontents'

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1602] [Fixed Issues]
After applying the rAudio 20240218 update to a known working system, I noticed the following in the update log:

![image](https://github.com/rern/rAudio/assets/76575081/5d225a13-cf42-4899-b716-0b72c04dcefc)

I then tried to play a webradio station and discovered that the content would not play (later found that this behavior also applies to playing album content).

When entering the Player menu, the flashing raudio logo is displayed and will not proceed further.  Refreshing web browser then successfully displays  the play screen.

Setting the Audio - I2s device to none with reboot seems to cleanup the issue as I can then enter the player screen and webradio / album content can be played.

However, RE-enabling the HiFiBerry DAC+ Pro device re-introduces the behavior as explained above.

Also, exact same behavior for 3B+ Rev 1.3 and 3B Rev 1.2 machines.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by Sagnasty** [comment]
Some additional info:
If you enter the player menu using either the onboard HDMI connection or firefox web browser (Windows PC) you get the perpetual flashing raudio logo.

If you enter the player menu using smartphone, it will display the json error:
[json parse error.txt](https://github.com/rern/rAudio/files/14324485/json.parse.error.txt)

This sounds very similar  (if not the same issue) that @ceejayemm reported as #1594.

**Comment by rern** [comment]
Please try today's update, 20240219.

**Comment by Sagnasty** [comment]
Issue solved with rAudio 20240219 update and working fine on 3B+ Rev 1.3 and 3B Rev 1.2 machines.

Thanks @rern
